Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/fenicsproject/dolfinx:dev-env-real
```

Comment:
Why is this needed? We have similar Dockerfile here https://github.com/FEniCS/dolfinx/blob/master/docker/Dockerfile

Reply:
To have this (much shorter) `Dockerfile` is primarily for convenience, secondly a must-have for customised test env containers in the scenario that certain dolfin-x features are not (yet) in master.

Reply:
For such case you could just pull dev-env-real and build dolfinx inside the container. We can keep this bit here, but need to remember this code is not tested.